support-frontend/assets/pages/paper-subscription-landing/components/NewspapperProductTabsStyles.tsx

```diff
@@ -5,7 +5,7 @@ export const cardsContainer = css`
 	display: flex;
 	flex-direction: column;
 	background-color: ${palette.brand[400]};
-	padding: 0 ${space[3]}px ${space[6]}px;
+	padding: 0 ${space[5]}px ${space[6]}px;
```

Comment:
This seems to have had the effect of increasing the padding around the cards a bit on mobile, where we don't even render the carousel. I wonder if it should be from the size at which we render the carousel (which I think is tablet)? What do you reckon?